integration/test.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from retrying import retry
+import time
+import unittest
+import uuid
+
+class CookTest(unittest.TestCase):
+
+    @retry(stop_max_delay=120000, wait_fixed=5000)
+    def wait_for_job(self, job_id, status):
+        job = requests.get('%s/rawscheduler?job=%s' % (self.cook_url, job_id))
```

Comment:
We should use request's session http://docs.python-requests.org/en/master/user/advanced/ as it makes it easier to sub in configuration like kerberos.